--- project/vendor/stesie/phpcs-doctrine-annotation-rules/src/DoctrineAnnotationCodingStandard/Helper/TypeHelper.php	(date 1286705410000)
+++ project/vendor/stesie/phpcs-doctrine-annotation-rules/src/DoctrineAnnotationCodingStandard/Helper/TypeHelper.php	(date 1286705410000)
@@ -28,9 +28,9 @@
             return self::convertPrimitiveType($varTagContent, $namespace, $classMap);
         }
 
-        if (in_array('null', $parts)) {
+        if (in_array('NULL', $parts)) {
             $parts = array_filter($parts, function (string $value) {
-                return $value !== 'null';
+                return $value !== 'NULL';
             });
             return new NullableType(self::fromString(implode('|', $parts), $namespace, $classMap));
         }
