--- project/vendor/stesie/phpcs-doctrine-annotation-rules/src/DoctrineAnnotationCodingStandard/Sniffs/Commenting/AbstractDoctrineAnnotationSniff.php	(date 1286705410000)
+++ project/vendor/stesie/phpcs-doctrine-annotation-rules/src/DoctrineAnnotationCodingStandard/Sniffs/Commenting/AbstractDoctrineAnnotationSniff.php	(date 1286705410000)
@@ -220,12 +220,6 @@
             $stackPtr++;
         }
 
-        if ($tokens[$stackPtr]['type'] !== 'T_SEMICOLON') {
-            throw new ParseErrorException(
-                \sprintf('Parse error on line %d: after T_USE, T_SEMICOLON expected', $tokens[$stackPtr]['line'])
-            );
-        }
-
         $this->imports->add($alias, $use);
     }
