--- project/vendor/squizlabs/php_codesniffer/src/Reports/Full.php	(date 1286705410000)
+++ project/vendor/squizlabs/php_codesniffer/src/Reports/Full.php	(date 1286705410000)
@@ -74,7 +74,7 @@
             }
         }
 
-        $file       = $report['filename'];
+        $file       = str_replace(getcwd(), '', $report['filename']);
         $fileLength = strlen($file);
         $maxWidth   = max(($fileLength + 6), ($maxErrorLength + $paddingLength));
         $width      = min($width, $maxWidth);
