--- project/vendor/infection/infection/src/Process/Builder/SubscriberBuilder.php	(date 1286705410000)
+++ project/vendor/infection/infection/src/Process/Builder/SubscriberBuilder.php	(date 1286705410000)
@@ -233,8 +233,6 @@
 
     private function shouldSkipProgressBars(): bool
     {
-        return $this->input->getOption('no-progress')
-            || getenv('CI') === 'true'
-            || getenv('CONTINUOUS_INTEGRATION') === 'true';
+        return $this->input->getOption('no-progress');
     }
 }
