--- project/vendor/infection/infection/src/Process/Listener/MutantCreatingConsoleLoggerSubscriber.php	(date 1286705410000)
+++ project/vendor/infection/infection/src/Process/Listener/MutantCreatingConsoleLoggerSubscriber.php	(date 1286705410000)
@@ -40,6 +40,7 @@
 use Infection\Events\MutantsCreatingFinished;
 use Infection\Events\MutantsCreatingStarted;
 use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Output\NullOutput;
 use Symfony\Component\Console\Output\OutputInterface;

 /**
@@ -57,11 +58,16 @@
      */
     private $progressBar;

+    /**
+     * @var int
+     */
+    private $counter = 1;
+
     public function __construct(OutputInterface $output)
     {
         $this->output = $output;

-        $this->progressBar = new ProgressBar($this->output);
+        $this->progressBar = new ProgressBar(new NullOutput());
         $this->progressBar->setFormat('Creating mutated files and processes: %current%/%max%');
     }

@@ -76,12 +82,30 @@

     public function onMutantsCreatingStarted(MutantsCreatingStarted $event): void
     {
-        $this->output->writeln(['']);
+        echo PHP_EOL;
         $this->progressBar->start($event->getMutantCount());
     }

     public function onMutantCreated(MutantCreated $event): void
     {
+        $numberLength = max(3, strlen((string) $this->progressBar->getMaxSteps()));
+        $maximumWidth = 50;
+
+        $message      = sprintf(
+            sprintf(' %%%sd / %%%sd (%%3d%%%%)', $numberLength, $numberLength),
+            $this->counter,
+            $this->progressBar->getMaxSteps(),
+            round(($this->counter / $this->progressBar->getMaxSteps()) * 100)
+        );
+
+        echo '.';
+        echo $this->counter % $maximumWidth === 0 ? sprintf('%s%s', $message, PHP_EOL) : '';
+
+        if ($this->progressBar->getMaxSteps() == $this->counter) {
+            echo sprintf('%s%s%s', str_repeat(' ', $maximumWidth - $this->counter % $maximumWidth), $message, PHP_EOL);
+        }
+
+        $this->counter++;
         $this->progressBar->advance();
     }
