--- backend/vendor/infection/infection/src/Process/Listener/CleanUpAfterMutationTestingFinishedSubscriber.php	(date 1286705410000)
+++ backend/vendor/infection/infection/src/Process/Listener/CleanUpAfterMutationTestingFinishedSubscriber.php	(date 1286705410000)
@@ -36,9 +36,7 @@
 
     public function getSubscribedEvents(): array
     {
-        return [
-            MutationTestingFinished::class => [$this, 'onMutationTestingFinished'],
-        ];
+        return [];
     }
 
     public function onMutationTestingFinished(MutationTestingFinished $event): void
