--- project/vendor/infection/infection/src/Performance/Listener/PerformanceLoggerSubscriber.php	(date 1286705410000)
+++ project/vendor/infection/infection/src/Performance/Listener/PerformanceLoggerSubscriber.php	(date 1286705410000)
@@ -69,15 +69,8 @@
 
     public function onApplicationExecutionFinished(): void
     {
-        $time = $this->timer->stop();
+        $this->timer->stop();
 
-        $this->output->writeln([
-            '',
-            sprintf(
-                'Time: %s. Memory: %s',
-                $this->timeFormatter->toHumanReadableString($time),
-                $this->memoryFormatter->toHumanReadableString(memory_get_peak_usage(true))
-            ),
-        ]);
+        $this->output->writeln('');
     }
 }
