--- project/vendor/infection/infection/src/Process/Listener/MutationTestingConsoleLoggerSubscriber.php	(date 1286705410000)
+++ project/vendor/infection/infection/src/Process/Listener/MutationTestingConsoleLoggerSubscriber.php	(date 1286705410000)
@@ -102,16 +102,6 @@
     public function onMutationTestingFinished(MutationTestingFinished $event): void
     {
         $this->outputFormatter->finish();
-
-        if ($this->showMutations) {
-            $this->showMutations($this->metricsCalculator->getEscapedMutantProcesses(), 'Escaped');
-
-            if ($this->output->getVerbosity() > OutputInterface::VERBOSITY_NORMAL) {
-                $this->showMutations($this->metricsCalculator->getNotCoveredMutantProcesses(), 'Not covered');
-            }
-        }
-
-        $this->showMetrics();
     }
 
     /**
