--- project/vendor/phpunit/phpunit/src/TextUI/TestRunner.php	(date 1286705410000)
+++ project/vendor/phpunit/phpunit/src/TextUI/TestRunner.php	(date 1286705410000)
@@ -268,10 +268,6 @@
             }
         }

-        $this->printer->write(
-            Version::getVersionString() . "\n"
-        );
-
         self::$versionStringPrinted = true;

         if ($arguments['verbose']) {
@@ -497,8 +493,6 @@
             }
         }

-        $this->printer->write("\n");
-
         if (isset($codeCoverage)) {
             $result->setCodeCoverage($codeCoverage);

@@ -542,46 +536,26 @@

         if (isset($codeCoverage)) {
             if (isset($arguments['coverageClover'])) {
-                $this->printer->write(
-                    "\nGenerating code coverage report in Clover XML format ..."
-                );
-
                 try {
                     $writer = new CloverReport;
                     $writer->process($codeCoverage, $arguments['coverageClover']);
-
-                    $this->printer->write(" done\n");
                     unset($writer);
                 } catch (CodeCoverageException $e) {
-                    $this->printer->write(
-                        " failed\n" . $e->getMessage() . "\n"
-                    );
+
                 }
             }

             if (isset($arguments['coverageCrap4J'])) {
-                $this->printer->write(
-                    "\nGenerating Crap4J report XML file ..."
-                );
-
                 try {
                     $writer = new Crap4jReport($arguments['crap4jThreshold']);
                     $writer->process($codeCoverage, $arguments['coverageCrap4J']);
-
-                    $this->printer->write(" done\n");
                     unset($writer);
                 } catch (CodeCoverageException $e) {
-                    $this->printer->write(
-                        " failed\n" . $e->getMessage() . "\n"
-                    );
+
                 }
             }

             if (isset($arguments['coverageHtml'])) {
-                $this->printer->write(
-                    "\nGenerating code coverage report in HTML format ..."
-                );
-
                 try {
                     $writer = new HtmlReport(
                         $arguments['reportLowUpperBound'],
@@ -593,31 +567,19 @@
                     );

                     $writer->process($codeCoverage, $arguments['coverageHtml']);
-
-                    $this->printer->write(" done\n");
                     unset($writer);
                 } catch (CodeCoverageException $e) {
-                    $this->printer->write(
-                        " failed\n" . $e->getMessage() . "\n"
-                    );
+
                 }
             }

             if (isset($arguments['coveragePHP'])) {
-                $this->printer->write(
-                    "\nGenerating code coverage report in PHP format ..."
-                );
-
                 try {
                     $writer = new PhpReport;
                     $writer->process($codeCoverage, $arguments['coveragePHP']);
-
-                    $this->printer->write(" done\n");
                     unset($writer);
                 } catch (CodeCoverageException $e) {
-                    $this->printer->write(
-                        " failed\n" . $e->getMessage() . "\n"
-                    );
+
                 }
             }

@@ -643,20 +605,12 @@
             }

             if (isset($arguments['coverageXml'])) {
-                $this->printer->write(
-                    "\nGenerating code coverage report in PHPUnit XML format ..."
-                );
-
                 try {
                     $writer = new XmlReport(Version::id());
                     $writer->process($codeCoverage, $arguments['coverageXml']);
-
-                    $this->printer->write(" done\n");
                     unset($writer);
                 } catch (CodeCoverageException $e) {
-                    $this->printer->write(
-                        " failed\n" . $e->getMessage() . "\n"
-                    );
+
                 }
             }
         }
